website/docs/docs/configure-dbt-extension.md

```diff
@@ -0,0 +1,207 @@
+2. Ensure you have a dbt [`profiles.yml` file](/docs/core/connect-data-platform/profiles.yml). This file defines your data warehouse connection. If you don't have one, you'll go through the dbt configuration processes to set it up. 
+3. Once you've set this up or checked it, validate it by running `dbt debug`.
+4. Add a `dbt_cloud.yml` file by going to the <Constant name="dbt_platform" /> project:
+   - **Your profile** -> **VS Code Extension** -> **Download credentials**. 
+   - Download the `dbt_cloud.yml` file with your [**Personal access Token (PAT)**](/docs/dbt-cloud-apis/user-tokens) included. This then connects the extension to <Constant name="dbt_platform" /> and enables platform features such as <Constant name="mesh" /> and deferral.
+   - Check the `project_id` in your `dbt_project.yml` file matching the project you're working on.
+5. Confirm connection from your workstation. Your local computer connects directly to your data warehouse and Git.  
+   - <Constant name="dbt_platform" /> users: Ensure your laptop/VPN is allowed; <Constant name="dbt_platform" /> IPs no longer apply. Check with your admin if you have any issues.
+   - <Constant name="core" /> users: This has likely already been configured.
+6. (Optional) If your project uses environment variables, find your environment variables and [set them](#set-environment-variables-locally) in VS Code or Cursor.
+   - <Constant name="dbt_platform" /> users: Copy any environment variables from **Deploy → Environments** **→ Environment variables** tab in <Constant name="dbt_platform" />.  Masked secrets will appear hidden, speak with your admin to get those values.  
+   - <Constant name="core" /> users: You'll most likely have defined your own [`env_var()` references](/reference/dbt-jinja-functions/env_var)  locally if your project uses them.
+
+## Set environment variables locally
+
+The extension and CLI uses environment variables for authentication and configuration. The following table shows the different options and when to use them:
+
+| Location | Affects | Session state | When to use |
+|-----------|----------|-----------|-----------|
+| [**Shell profile** ](#configure-at-the-os--shell-level)| Terminal  | ✅ Permanent | Variables remain active globally and available across terminal sessions.|
+| [**VS Code/Cursor settings**](#configure-in-the-vs-code-extension-settings) | Extension menus + <Term id="lsp" /> | ✅ Per VS Code/Cursor profile | Editor-only workflows using the extension menu actions. |
+| [**One terminal session**](#configure-in-the-terminal-session)  | Current terminal only | ❌ Temporary | One off quick testing. |
+
+:::tip 
+If you want to use both the VS Code extension menus and terminal to run dbt commands, define your variables in both the `shell` profile so they remain active globally and in VS Code/Cursor settings if you use extension menu actions.
```

Comment:
Looks like we're missing a second option?
"...in both the `shell` profile" and where else?

Reply:
this is a mistake, we only want to sugg .env / vs code settings and shell.